sunkit_pyvista/tests/test_plotting.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pfsspy
+import pytest
+import pyvista
+from pfsspy import tracing
+from pfsspy.sample_data import get_gong_map
+
+import astropy.constants as const
+import astropy.units as u
+import sunpy.data.test as test
+import sunpy.map as smap
+from astropy.coordinates import SkyCoord
+from sunpy.coordinates import frames
+
+from sunkit_pyvista import SunpyPlotter
+
+pyvista.OFF_SCREEN = True
+
+
+@pytest.fixture
+def aia171_test_map():
+    return smap.Map(test.get_test_filepath('aia_171_level1.fits'))
+
+
+@pytest.fixture
+def plotter():
+    return SunpyPlotter()
+
+
+def test_plot_map_with_functionlity(aia171_test_map, plotter, verify_cache_image):
+    plotter.plot_map(aia171_test_map, clip_interval=(0, 99)*u.percent)
+    plotter.plot_solar_axis()
+
+    bottom_left = SkyCoord(30*u.deg, -10*u.deg,
+                           frame=frames.HeliographicStonyhurst,
+                           obstime=aia171_test_map.date)
+    plotter.plot_quadrangle(bottom_left=bottom_left, width=20*u.deg,
+                            height=60*u.deg, color='blue', line_width=5)
+
+    line = SkyCoord(lon=[180, 190, 200] * u.deg,
```

Comment:
Should this line be on the sun or just around?

Reply:
It is an arbitrary line, I didn't really think it had any use except to show it was possible, so I let it be. @dstansby Does the position of the line matter?

Reply:
I think at least it being near the sun so we could zoom in a little would be nicer.

Reply:
Alright, I shall make this change.